index.js

```diff
@@ -311,18 +311,57 @@ function cms(loopbackApplication, options) {
     });
   });
 
+  if (config.public.isUnsafeUpsert) {
```

Comment:
I've changed it to config.allowUnsafeUpsert as @csapinoso requested and also works either way.